README.md

```diff
@@ -3,8 +3,9 @@ GitHub Services
 
 This repository contains code to integrate GitHub.com with third party services.
 
-See the [Contributing Guidelines](https://github.com/github/github-services/blob/master/CONTRIBUTING.md)
-for instructions on contributing a service.
+Please note: GitHub will only accept pull requests to existing services that implement bug fixes or security improvements. We no longer accept feature changes to existing services.
```

Comment:
I would suggest dropping this under `Current Status`. That way the header becomes a link and we can send a direct message to incoming requests.